src/Constants.ts

```diff
@@ -8,6 +8,11 @@ export const CONFIG_FILE = 'Config.json';
 export const PMD_CATALOG_FILE = 'PmdCatalog.json';
 export const INTERNAL_ERROR_CODE = 500;
 
+export const PILOT_AVAILABILITY_BANNER= `As of April 27, 2022, v3.x of the scanner is available as an open pilot.
```

Comment:
@rmohan20 , @teresa-allen-sfdc , important note: Since I'm using `ux.warn()` to display the message, it will be preceded by "WARNING: ".
Is that okay? If not, we can go to `ux.log()`.

Reply:
Can it be a `info()`?

Reply:
There's no `info`, but we could do `ux.styledHeader()`.

Reply:
@rmohan20 @jfeingold35 Question: what is the one consistent term that we are using to call the feature in ALL docs? SFCA? scanner? analyzer? Since we rebranded to Salesforce Code Analyzer (SFCA), what stands out to me in this note is that "v3.x of the **scanner** is available ...".

Scanner to me is deprecated since we rebranded to "Code Analyzer" and we shouldn't use scanner anywhere. To me, it causes reader confusion to use the terms interchangeably. I vote for SFCA because that's the product/feature acronym. My second choice is analyzer.

**Edit:**

As of April 27, 2022, version 3.x of the Salesforce Code Analyzer (SFCA) is available as an open pilot.

Reply:
@teresa-allen-sfdc Great catch. Agree that we shouldn't be calling it "Scanner". "Code Analyzer" is the term we've been using.

Reply:
Yeah, we've been calling it Code Analayzer.